.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
I never use a .gitignore.
I try to be thoughful when adding files
Could you revert this change?

Reply:
I will revert it.

I mostly add it so that ‘git status’ only shows relevant stuff.